### DISCUSSION TITLE
Set body function for response based on mime type in Kong PDK

### DISCUSSION THREAD

**Discussion by anup-krai** [Kong/kong#10128] [Help]
Hi,
Is there any function available to set the response body based on the mime type in body_filter phase?

For request we have below function available -
[kong.service.request.set_body(args[, mimetype])](https://docs.konghq.com/gateway/latest/plugin-development/pdk/kong.service.request/#kongservicerequestset_bodyargs-mimetype)

Is something similar available for response also?

**Comment by StarlightIbuki** [comment]
Please referer: https://docs.konghq.com/hub/kong-inc/response-transformer/
